fact_extractor/test/unit/helperFunctions/test_file_system.py

```diff
@@ -16,13 +15,10 @@ def setUp(self):
     def tearDown(self):
         os.chdir(self.current_cwd)
 
-    def test_get_parent_dir(self):
-        self.assertEqual(get_parent_dir('/foo/bar/test'), '/foo/bar', 'parent directory')
-
     def check_correct_src_dir(self, working_directory):
         real_src_dir = get_src_dir()
         os.chdir(working_directory)
-        self.assertTrue(os.path.exists('{}/helperFunctions/fileSystem.py'.format(real_src_dir)), 'fileSystem.py found in correct place')
+        self.assertTrue(os.path.exists('{}/helperFunctions/file_system.py'.format(real_src_dir)), 'file_system.py found in correct place')
```

Comment:
maybe we could get rid of os.path here, too
e.g. `(Path(real_src_dir) / 'helperFunctions' / 'file_system.py').is_file()`